package/sbp_rtcm3_bridge/src/sbp_rtcm3.h

```diff
@@ -28,6 +29,8 @@
 #define MAX_SBP_PAYLOAD 255
 #define MAX_OBS_IN_SBP ((MAX_SBP_PAYLOAD - SBP_HEADER_SIZE) / SBP_OBS_SIZE)
 
+static bool debug = false;
```

Comment:
Does this work? Won't it just put a version of debug in each compilation unit - I think this needs to be extern'd.